includes/class-wc-payments-account.php

```diff
@@ -742,4 +747,61 @@ private function get_actioned_notes(): array {
 
 		return $wcpay_note_names;
 	}
+
+	/**
+	 * Checks to see if the account is eligible for Instant Deposits.
+	 *
+	 * @return bool
+	 */
+	public function is_instant_deposits_eligible(): bool {
+		$account = $this->get_cached_account_data();
+		if ( ! isset( $account['instant_deposits_eligible'] ) || ! $account['instant_deposits_eligible'] ) {
+			return false;
+		}
+
+		return true;
+	}
+	/**
+	 * Handles adding a note if the merchant is eligible for Instant Deposits.
+	 *
+	 * @return void
+	 */
+	public function handle_instant_deposits_inbox_note(): void {
```

Comment:
The `void` return type isn't valid for PHP 7.0 (at least that's what the linter says).